Self-Consistent-Field/RHF_Gradient.py

```diff
@@ -0,0 +1,176 @@
+"""
+This script calculates nuclear gradients of RHF using
+derivative integrals obtained from PSI4
+
+"""
+
+__authors__ = "Ashutosh Kumar"
+__credits__ = ["Ashutosh Kumar"]
+
+__copyright__ = "(c) 2014-2017, The Psi4NumPy Developers"
+__license__ = "BSD-3-Clause"
+__date__ = "2017-12-17"
```

Comment:
Any gradient citation that you can base these off of. @andysim might have ideas for a good paper as well.

Reply:
@andysim I mostly followed the code that you wrote in PSI4 for both gradients and hessians. Could you please let me know what reference did you follow ?

Reply:
[This](http://onlinelibrary.wiley.com/doi/10.1002/qua.560160825/abstract) is probably the best overview of the derivation, IMHO.

Reply:
Thanks @andysim

Reply:
Done!